cfpb/consumerfinance.gov
# Allow empty content on AbstractFilterablePages

Commit 2f7168072262c4943443f53aa3d8310853bd2925 made it possible to remove filterable list controls from a filterable page, but it still kept the existing requirement of a non-empty content field.

On our filterable page types (BrowseFilterablePage and SublandingFilterablePage) we might want to have an empty content field, if we only want a header and search results. This commit makes that possible by making the content field optional.

This work relates to internal D&CP#139.

## How to test this PR

A good page to test this on using a production dump is the Events page, locally at http://localhost:8000/admin/pages/1090/edit/. Try deleting the FilterableList module from the content page and note that the page preview still works and that the page can be saved or published.

## Checklist
